test_infra/test_files/30001/RUN1/Alignments/100/alignment_metadata.json

```diff
@@ -76,6 +44,7 @@
   },
   "tilt_offset": 0,
   "x_rotation_offset": 0,
-  "deposition_id": 301,
-  "last_updated_at": 1728675677
+  "deposition_id": 300,
+  "last_updated_at": 1728675677,
+  "tiltseries_id": 501
```

Comment:
`tiltseries_id` is not a field that exists in the data produced by our S3 ingestion process -- I think we should avoid adding arbitrary fields to our test data.